Makefile

```diff
@@ -1,4 +1,4 @@
-all: pep8 pylint tests build
+all: build
```

Comment:
all, should be all: pep8 pylint tests & build,
We can move build to the top so it will be the default target.